Source/WebCore/platform/KeyboardScrollingAnimator.cpp

```diff
@@ -212,12 +181,6 @@ bool KeyboardScrollingAnimator::beginKeyboardScrollGesture(ScrollDirection direc
     if (!scroll)
         return false;
 
-    auto scrollableDirections = scrollableDirectionsFromPosition(m_scrollableArea.scrollAnimator().currentPosition());
```

Comment:
The iOS code has a 

```
if (![_scrollable rubberbandableDirections].at(boxSide(scroll->direction)))
    return NO;
```

here. Not sure it's important, but I bet that's what this code intended to be doing.